pkgs/development/libraries/precice/default.nix

```diff
@@ -12,14 +12,21 @@ stdenv.mkDerivation rec {
   };
 
   preConfigure = ''
-    cmakeFlags="-DBUILD_SHARED_LIBS=ON -DPETSC=off"
+    cmakeFlags="-DBUILD_SHARED_LIBS=ON
+                -DPETSC=off
+                -DCMAKE_BUILD_TYPE=Debug
+                -DCMAKE_INSTALL_PREFIX=$out
+                "
   '';
-
   nativeBuildInputs = [ cmake gcc ];
   buildInputs = [ boost eigen libxml2 openmpi python2 python2Packages.numpy ];
```

Comment:
Is python3 still not supported?

Reply:
Hi there, it is now https://github.com/precice/precice/pull/481/